lib/tasks/grade.rake

```diff
@@ -10,5 +10,9 @@ task :grade, :token do |t, args|
   username = git_url.split(':')[1].split('/')[0]
   reponame =  git_url.split(':')[1].split('/')[1].sub(".git", "")
   sha = `git rev-parse --verify HEAD`.chomp
-  GradeRunner::Runner.new(config['project_token'], config['submission_url'], token, rspec_output_json, username, reponame, sha).process
+  if token.present?
+    GradeRunner::Runner.new(config['project_token'], config['submission_url'], token, rspec_output_json, username, reponame, sha, 'manual').process
+  else
+    puts "Token not available. Build not submitted to canvas. Please launch from canvas assignment and run rails grade as shown in resource page."
```

Comment:
I would say, "We couldn't find your access token, so we couldn't record your grade. Please click on the assignment link again and run the `rails grade ...` command shown there."